### PR TITLE
fix#4805: in single and double quotes

### PR DESCRIPTION
**Pull request by hajjem-ayoub**
fix for #4805

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -28,7 +28,7 @@
...
+    "postinstall": "node --eval \"if (require('./package.json').name === 'coffee-script') { var red, yellow, cyan, reset; red = yellow = cyan = reset = ''; if (!process.env.NODE_DISABLE_COLORS) { red = '\\x1b[31m'; yellow = '\\x1b[33m'; cyan = '\\x1b[36m'; reset = '\\x1b[0m'; } console.warn(red + 'CoffeeScript has moved!' + reset + ' Please update references to ' + yellow + '\\\"coffee-script\\\"' + reset + ' to use ' + yellow + '\\\"coffeescript\\\"' + reset + ' (no hyphen) instead.'); console.warn('Also, a new major version has been released under the ' + yellow + 'coffeescript' + reset + ' name on NPM. This new release targets modern JavaScript, with minimal breaking changes. Learn more at ' + cyan + 'http://coffeescript.org' + reset + '.'); console.warn(''); }\"",
```
Comment (vendethiel) [inline package.json]: have you tried `\\\"coffee-script\\\"` and confirm it works on both platforms?

Comment (xameeramir) [reply package.json]: @hajjem-ayoub Does it works on your local with this fix?

Comment (hajjem-ayoub) [reply package.json]: yes tried it only with windows VM, i can also check on mac

Comment (rossanmol) [reply package.json]: This shouldn't work.

Comment (hajjem-ayoub) [reply package.json]: i also tried to npm link it in mac and it seems to work as well, it shows the message correctly at least

Comment (hajjem-ayoub) [reply package.json]: @rossanmol just to debug it, can you tell me how did you test it?

**Comment by rossanmol** [comment]
What about this solution:

`node --eval "if (require(`./package.json`).name === `coffee-script`) { var red, yellow, cyan, reset; red = yellow = cyan = reset = ``; if (!process.env.NODE_DISABLE_COLORS) { red = `\\x1b[31m`; yellow = `\\x1b[33m`; cyan = `\\x1b[36m`; reset = `\\x1b[0m`; } console.warn(red + `CoffeeScript has moved!` + reset + ` Please update references to ` + yellow + `\\\"coffee-script\\\"` + reset + ` to use ` + yellow + `\\\"coffeescript\\\"` + reset + ` (no hyphen) instead.`); console.warn(`Also, a new major version has been released under the ` + yellow + `coffeescript` + reset + ` name on NPM. This new release targets modern JavaScript, with minimal breaking changes. Learn more at ` + cyan + `http://coffeescript.org` + reset + `.`); console.warn(``); }"`

**Comment by rossanmol** [comment]
What about this solution:

> node --eval "if (require(`./package.json`).name === `coffee-script`) { var red, yellow, cyan, reset; red = yellow = cyan = reset = ``; if (!process.env.NODE_DISABLE_COLORS) { red = `\\x1b[31m`; yellow = `\\x1b[33m`; cyan = `\\x1b[36m`; reset = `\\x1b[0m`; } console.warn(red + `CoffeeScript has moved!` + reset + ` Please update references to ` + yellow + `\\\"coffee-script\\\"` + reset + ` to use ` + yellow + `\\\"coffeescript\\\"` + reset + ` (no hyphen) instead.`); console.warn(`Also, a new major version has been released under the ` + yellow + `coffeescript` + reset + ` name on NPM. This new release targets modern JavaScript, with minimal breaking changes. Learn more at ` + cyan + `http://coffeescript.org` + reset + `.`); console.warn(``); }"

File: package.json (hunk 1 above)
Comment (rossanmol) [reply package.json]: run `node --eval` in cmd

**Comment by vendethiel** [comment]
@rossanmol people using 1.x not all have access to ES6+ features. That's kinda the point of the 1.x branch.

**Comment by hajjem-ayoub** [comment]
@rossanmol you cannot use ` you can only use ' or " because es6 not everywhere, but i will try to look for windows 10 to check, but can you tell me how you tested??
did you pull the code and npm linked it??

**Comment by rossanmol** [comment]
@hajjem-ayoub run the following in cmd:

```
node --eval \"if (require('./package.json').name === 'coffee-script') { var red, yellow, cyan, reset; red = yellow = cyan = reset = ''; if (!process.env.NODE_DISABLE_COLORS) { red = '\\x1b[31m'; yellow = '\\x1b[33m'; cyan = '\\x1b[36m'; reset = '\\x1b[0m'; } console.warn(red + 'CoffeeScript has moved!' + reset + ' Please update references to ' + yellow + '\\\"coffee-script\\\"' + reset + ' to use ' + yellow + '\\\"coffeescript\\\"' + reset + ' (no hyphen) instead.'); console.warn('Also, a new major version has been released under the ' + yellow + 'coffeescript' + reset + ' name on NPM. This new release targets modern JavaScript, with minimal breaking changes. Learn more at ' + cyan + 'http://coffeescript.org' + reset + '.'); console.warn(''); }\"
```

I didn't try linking it.

**Comment by hajjem-ayoub** [comment]
@rossanmol  yes but than you don't have process.env.NODE_DISABLE_COLORS

**Comment by rossanmol** [comment]
@hajjem-ayoub it fails because of the if statement, same as before.

**Comment by rossanmol** [comment]
@hajjem-ayoub placing the command into a file might be a solution too.

**Comment by hajjem-ayoub** [comment]
just  pushing one like that now :D

**Comment by hajjem-ayoub** [comment]
@rossanmol  and for coloring i used chalk

**Comment by vendethiel** [comment]
@hajjem-ayoub if anything, you missed a git add.

**Comment by hajjem-ayoub** [comment]
@vendethiel  sorry for that, i amended my commit and push forced again

**Comment by hajjem-ayoub** [comment]
@vendethiel done with this and it looks like working fine, i just had to add chalk to dependancies if that's ok

**Comment by vendethiel** [comment]
I don't think coloring is important enough to warrant adding a dep.

**Comment by GeoffreyBooth** [comment]
Yes, we don’t have dependencies. I think #4807 is closer to what we need to do.

**Pull request closed without merging**